README.md

```diff
@@ -135,26 +171,32 @@ Which method you use depends on how you've configured your CLI options. Please r
 
 #### Component Library (recommended)
 
-| Prop          | Type   | Description                                                                                                                                                                                                                                             |
+| Prop          | Type   | Description |
 | ------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `componentId` | string | The API ID of a component in your component library. If a `variant` prop is passed to an ancestor `DittoProvider`, will attempt to display the specified variant's value for the passed `componentId`; otherwise, will default to displaying base text. |
-
+| `variables` | object (optional) | This prop contains a map of all the variable key-value pairs you wish to pass into your text  |
```

Comment:
In the rich text diff, this is showing up as a single row. Is there a way to make it 2 rows? 

![image](https://user-images.githubusercontent.com/13909354/150185643-07004201-d1a3-4e42-849f-c357e47e8569.png)

Reply:
not sure what tool you were using but there should be a line currently: 
![image](https://user-images.githubusercontent.com/7476817/150188300-e0904734-f333-47d4-94de-04dff3b7da30.png)